polygeist/tools/cgeist/Lib/clang-mlir.cc

```diff
@@ -5773,8 +5846,11 @@ class MLIRAction : public clang::ASTFrontendAction {
   }
 };
 
-mlir::func::FuncOp MLIRScanner::EmitDirectCallee(const FunctionDecl *FD) {
-  return Glob.GetOrCreateMLIRFunction(FD, true);
+mlir::FunctionOpInterface
+MLIRScanner::EmitDirectCallee(const FunctionDecl *FD) {
```

Comment:
I think we should pass to this member function the enumerator that identifies whether the function should be created in the host or device module.

Reply:
As this one is not used, I wasn't sure whether to add it or not. I think it makes sense. Added.